src/DynamoCoreWpf/Controls/DynamoTextBox.cs

```diff
@@ -436,8 +445,8 @@ private CustomPopupPlacement[] PlacementCallback(Size popup, Size target, Point
 
             // Count width.
             double x = 0;
-            x = WpfUtilities.FindUpVisualTree<SearchView>(this.PlacementTarget).ActualWidth
-                + gap * 2 + targetLocation.X * (-1);
+            x = (WpfUtilities.FindUpVisualTree<SearchView>(this.PlacementTarget).ActualWidth
+                + gap * 2 + targetLocation.X * (-1)) * xfactor;
```

Comment:
I think the intention here was to thicken the gap, so yes we should keep *2, otherwise it will become thinner on both kinds of screens.